include/metar.h

```diff
@@ -42,7 +42,9 @@ typedef unsigned short int MDSP_BOOL;
 #define MAX_CLOUD_GROUPS 6
 #define MAX_SURFACE_OBSCURATIONS 6
 #define MAX_PARTIAL_OBSCURATIONS 2
- 
+
+typedef enum distance_unit { FT, METERS } Distance_Unit;
```

Comment:
For consistency, maybe change `FT` to be `FEET` since you spell out METERS?

Also I wonder if it would be better to use longer names for these enums to avoid possibly name collisions with people's programs? Maybe something like DIST_FEET and DIST_METERS?